### ISSUE TITLE
Migrate code to publish using github pages

### ISSUE THREAD

**Issue by larryms** [Microsoft/MTC_ContainerCamp#74]

**Comment by larryms** [comment]
sample at my fork of the repo:  https://larryms.github.io/MTC_ContainerCamp/

**Comment by amazaheri** [comment]
@larryms still wanna work on this?

**Comment by larryms** [comment]
yes, definitely.  The complication would be, there's no way to stage the changes in a branch to preview it, as GitHub pages only work against one and only one branch (e.g, master).    
As a workaround, we could fork the repo and then stage it in the fork.

**Comment by larryms** [comment]
We can also look at DocFX as an option:  https://dotnet.github.io/docfx/
